openstudioreleases/README.md

```diff
@@ -53,7 +113,13 @@ service cloud.firestore {
 
 The data is stored in the `users` collection.
 
-The script below will get all users from the database and save to a json file
+The script below will get all users from the database and save to a json file.
+
+For it to work, you need the `firebase-service-account.json` with the private key next to this README.md (Ask the maintainers).
+
+To generate one you will need to go to the [Firebase project Service Accounts settings](https://console.firebase.google.com/u/0/project/osc-downloads/settings/serviceaccounts/adminsdk)
+
+Generate a new private key, and save it to `firebase-service-account.json`. **Note: as soon as you generate a new private key, the old key will stop working, so ask the maintainers to provide it instead**
```

Comment:
@macumber please review this carefully... (also, do you have that json file to share with me?)